### ISSUE TITLE
Create regression test suites for birth section

### ISSUE THREAD

**Issue by rikukissa** [opencrvs/opencrvs-core#6964]
### Description

Using test cases from [here](https://docs.google.com/spreadsheets/d/1Ppf29wo0whdscmoeE8m4G3wCIVeorfHP/edit#gid=648252876), write Cypress tests in OpenCRVS Farajaland repository for all test cases under **Birth**.

Test need to be written as per the testing guidelines [here](https://www.notion.so/opencrvs/E2E-testing-framework-5d14d933c94e4b869220fd52035f60db?pvs=4).

### Dev tasks

- [x] Familiarise yourself with existing test code and helpers related to birth
- [x] Create necessary test files for each test case under `cypress/e2e/test-cases/birth`
- [x] Submit a PR after finishing the first test file
- [x] Continue work on other login tests until the login section is covered

**Comment by SyedaAfrida** [comment]
1. Uploading supporting documents are not done in automation scripts
2. In Declaration review page UI not showing properly

![Image](https://github.com/user-attachments/assets/d4f943f8-a98b-484a-8205-6f2a3d9759ae)

**Comment by jamil314** [comment]
@SyedaAfrida The ui issue is most likely a playwright bus and it does affect any functionality. Can it be ignored for now?

**Comment by SyedaAfrida** [comment]
@jamil314 yes this can be ignored. But for declaration 3 &7, for uploading supporting document, please follow this testcase "Title: Validate Attaching supporting documents page" testcase 7 in birth section.
